tests/test_modeling_bert.py

```diff
@@ -260,6 +260,61 @@ def create_and_check_model_for_causal_lm_as_decoder(
         )
         self.parent.assertEqual(result.logits.shape, (self.batch_size, self.seq_length, self.vocab_size))
 
+    def create_and_check_decoder_model_past_large_inputs(
```

Comment:
the test is already very nice. I just made the other "past" tests a bit more aggressive: https://github.com/huggingface/transformers/blob/e9d77ccd5a783dba83361bb46d3c615e6acadaee/tests/test_modeling_bart.py#L159 -> could we maybe add the `attention_mask` here as well to be sure it's correctly handled by the caching